bluemira/utilities/tools.py

```diff
@@ -299,6 +300,15 @@ def cross(
 # =====================================================
 
 
+def floatify(x: npt.ArrayLike):
+    """
+    Avoid numpy warnings for float(x) for >0 rank scalars
```

Comment:
Maybe a slightly more explicate explanation

"Converts the np array or float into a float by returning the first element or the element itself"